### DISCUSSION TITLE
Help me to connect Emailjs with brevo.

### DISCUSSION THREAD

**Discussion by noordeen123** [emailjs-com/help#170] [Q&A]
I have added my brevo user and key to the email js plat form but whenever i send an email brevo throws this error 
"Sending has been rejected because the sender you used 827553007@smtp-brevo.com is not valid. Validate your sender or authenticate your domain" but i have authenticated my domain and added sender email.

**Comment by xr0master** [comment]
Hi,
During service creation or template testing?
Perhaps, the easiest option to resolve it is to add your user address on the EmailJS dashboard to verify as a sender in Brevo.
